plugins/convert.py

```diff
@@ -31,17 +31,26 @@ def convert(inp, bot=None):
     parts = input.split()
 
     try:
-        ffrom = parts[0]
-        to = parts[1]
-    except IndexError:
+        parts.remove('TO')
+    except ValueError:
+        pass
+
+    amount = None
+    for x in parts:
+        try:
+            amount = float(x)
+            parts.remove(x)
+        except ValueError:
+            pass
+    if (amount == None):
+        amount = 1.0
+
+    # this, or try/except
+    if (len(parts) != 2):
```

Comment:
Actually, can you keep the original try except please. Catch the index error, don't care if they've got other stuff written in the arguments